client/my-sites/marketing/connections/service-tip.jsx

```diff
@@ -16,14 +20,34 @@ import { Component } from 'react';
  *
  * @type {string[]}
  */
-const SERVICES_WITH_TIPS = [ 'facebook', 'twitter', 'instagram', 'google_plus' ];
+const SERVICES_WITH_TIPS = [ 'twitter', 'instagram', 'google_plus' ];
+/**
+ * List of services that should only have tips on Jetpack sites.
+ *
+ * @type {string[]}
+ */
+const JETPACK_ONLY_TIPS = [ 'facebook' ];
```

Comment:
More nitpicks! As this is always concatenated onto `SERVICES_WITH_TIPS` could we define this as:
```suggestion
const JETPACK_SERVICES_WITH_TIPS = SERVICES_WITH_TIPS.concat( [ 'facebook' ] );
```

Reply:
Good idea!